src/container/container.ts

```diff
@@ -494,19 +494,16 @@ class Container implements interfaces.Container {
     }
 
     private _getContainerModuleHelpersFactory() {
-        const _bindOrRebindWithModuleId = (
-            moduleId: interfaces.ContainerModuleBase["id"],
-            serviceIdentifier: interfaces.ServiceIdentifier<any>,
-            bind:boolean) => {
-            const bindingToSyntax = this[bind ? "bind" : "rebind"](serviceIdentifier);
-            (bindingToSyntax as any)._binding.moduleId = moduleId;
-            return bindingToSyntax;
-        };
 
-        const getBindFunction = (moduleId: interfaces.ContainerModuleBase["id"]) =>
-            (serviceIdentifier: interfaces.ServiceIdentifier<any>) => {
-                return _bindOrRebindWithModuleId(moduleId, serviceIdentifier,true);
-            };
+        const createBind = (bindOrRebind: "bind"|"rebind") =>
+            (moduleId: interfaces.ContainerModuleBase["id"]) =>
+                (serviceIdentifier: interfaces.ServiceIdentifier<any>) => {
+                    const bindingToSyntax = this[bindOrRebind](serviceIdentifier);
+                    (bindingToSyntax as any)._binding.moduleId = moduleId;
+                    return bindingToSyntax;
+                };
```

Comment:
We should not refactor this. It's a false positive. Even if code climate suggests a refactor there is no way to improve the code

Reply:
this is gonna be hard to type :D when we do move to stricter ts rules and eslint

Reply:
If you are not happy with some of these refactors can I leave it for you to revert them ?

Reply:
Sure, allow me to handle it :). It's still a great job @tonyhallett, thank you so much!

Reply:
No no this is fine. I am just writing this so we know when we are gonna do the upgrades. I didn't want to sound rude or anything, just wanted to draw attention to those things

Reply:
@PodaruDragos 
No offence taken !